docs/guides/job-limits.mdx

```diff
@@ -7,10 +7,11 @@ description: Job limits imposed by the job validation service for jobs sent to I
 
 When you send a job to an IBM&reg; QPU, it is first sent to the job validation service.  This service tries to ensure that the job will be able to run on the QPU so you don't have to wait for it to go through the queue and then have the job fail.  These checks include enforcing the described limits.
 
-<Admonition type="note">
-Certain primitive options increase the circuit size.  The described limits are checked _after_ the expected increase in circuit size. In particular, these options increase circuit size:
+<Admonition type="note" title="Notes">
+- Certain primitive options increase the circuit size.  The described limits are checked _after_ the expected increase in circuit size. In particular, these options increase circuit size:
   - Dynamical decoupling and gate-folding ZNE introduce additional gates that are included in the instructions for the [Maximum number of low-level instructions per qubit](#per-qubit) limit.
   - Gate-folding ZNE introduces additional two-qubit gates relevant to the [Maximum number of two-qubit gates per job](#2-qubit-limit) limit. The number of two-qubit gates is multiplied by the sum of noise factors requested in gate-folding ZNE.
+- The `backend.configuration()` fields, `max_shots` and `max_experiments`, are no longer relevant and will be removed in the near future.
```

Comment:
```suggestion
- The `backend.configuration()` fields `max_shots` and `max_experiments` are no longer relevant and will be removed in the near future.
```

Reply:
There are more fields in `backend.configuration()` than those. Although I think we should word it as something like

> The limits reported by the `backend.configuration()` fields `max_shots` and `max_experiments` are no longer relevant or enforced. These fields will be removed in the near future.

to make it clearer.